src/app/components/pop-up-how-to-play/pop-up-how-to-play.component.scss

```diff
@@ -14,75 +14,63 @@
   flex-direction: column;
   align-items: center;
   transition: transform 0.5s ease;
-}
-
-.pop-up-nav {
-  margin-bottom: 24px;
-  display: flex;
-  width: 100%;
-  justify-content: space-between;
-}
-
-.pop-up-nav img {
-  width: 24px ;
-  mix-blend-mode: difference;
-}
-
-.pop-up-nav img:hover {
-  mix-blend-mode: color-dodge;
-}
-
-.pop-up-nav img:active {
-  mix-blend-mode: exclusion;
-}
-
-.pop-up-nav h1 {
-  font-size: 24px;
-  font-weight: 500;
-  display: flex;
-  margin: 18px 0;
-}
-
-.pop-up-main {
-  width: 100%;
-}
-
-.pop-up-main p {
-  font-size: 14px;
-  font-weight: 400;
-  line-height: 24px;
-}
-
-.pop-up-main span {
-  color: $error-text-color;
-}
-
-.pop-up-footer {
-  border: 1px solid #373744;
-  border-radius: 8px;
-  width: 100%;
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  gap: 8px;
-}
-
-.pop-up-footer p {
-  margin: 0px;
-  padding: 16px 0;
+  &__nav {
```

Comment:
Colocar uma linha em branco entre as classes para melhorar a legibilidade.

Reply:
Tentei inserir as linhas em branco entre o métodos mas o Lint não está permitindo.

Trailing spaces not allowed.eslint[no-trailing-spaces](https://eslint.org/docs/latest/rules/no-trailing-spaces)
Delete ··eslint[prettier/prettier](https://github.com/prettier/eslint-plugin-prettier#options)